### PR TITLE
Add localization messages for 'fr' locale

### PR DESCRIPTION
**Pull request by perrich**
Hi,

As done for some other language I've added a french translation based on the last version of en.json file.

Hope it can help.

### CODE REVIEW

**Review by AlekseyMartynov — COMMENTED** [review]
Hi
Thank you very much.
Creating a database of user translations is in our plans. Currently we are not quite sure whether it will be a part of this or another, "contrib"-like repository. I'll keep this PR open until we make a decision on this.

**Comment by perrich** [comment]
Ok, thanks.

File: js/localization/messages/fr.json [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        "dxCalendar-ariaWidgetName": "Calendrier",
+
+        "dxColorView-ariaRed": "Rouge",
+        "dxColorView-ariaGreen": "Vert",
+        "dxColorView-ariaBlue": "Bleu",
+        "dxColorView-ariaAlpha": "Transparence",
+        "dxColorView-ariaHex": "Code couleur",
+
+        "vizExport-printingButtonText": "Imprimer",
+        "vizExport-titleMenuText": "Exporterg/Imprimer",
```
Comment (BingoRUS) [inline js/localization/messages/fr.json]: Hi, @perrich!
"Exporterg" - is it the right translation? It seems there is a typo...

Comment (perrich) [reply js/localization/messages/fr.json]: Sorry for the delay, I've lost the notification :(

You're right. It's a typing issue, the correct translation is "Exporter/Imprimer".

**Comment by perrich** [comment]
I will check for newly added messages and update the PR.

**Review by BingoRUS — APPROVED** [review]
👌

**Comment by BingoRUS** [comment]
@perrich Thank you! I'm going to merge this pull request now. French localization will be available with the next 17.1 minor release inside npm, Bower and Nuget packages as well as on CDN.
Also I will ask you to help us with some new localization messages what we added in the latest (17.2) release if you don't mind.

**Pull request merged**